tensorboard/backend/application.py

```diff
@@ -104,12 +108,32 @@ def standard_tensorboard_wsgi(flags, plugin_loaders, assets_zip_provider):
       tensor_size_guidance=tensor_size_guidance_from_flags(flags),
       purge_orphaned_data=flags.purge_orphaned_data,
       max_reload_threads=flags.max_reload_threads)
-  db_module, db_connection_provider = get_database_info(flags.db)
+  loading_multiplexer = multiplexer
+  reload_interval = flags.reload_interval
+  db_uri = flags.db
+  # For DB import mode, create a DB file if we weren't given one.
+  if flags.db_import and not flags.db:
+    tmpdir = tempfile.mkdtemp(prefix='tbimport')
+    atexit.register(shutil.rmtree, tmpdir)
+    db_uri = 'sqlite:%s/tmp.sqlite' % tmpdir
+  db_module, db_connection_provider = get_database_info(db_uri)
+  if flags.db_import:
+    # DB import mode.
+    if db_module != sqlite3:
+        raise ValueError('--db_import is only compatible with sqlite DBs')
+    tf.logging.info('Importing logdir into DB at %s', db_uri)
+    loading_multiplexer = db_import_multiplexer.DbImportMultiplexer(
+        db_connection_provider=db_connection_provider,
+        purge_orphaned_data=flags.purge_orphaned_data,
+        max_reload_threads=flags.max_reload_threads)
+  elif flags.db:
+    # DB read-only mode, never load event logs.
+    reload_interval = -1
```

Comment:
nit: seems like this should be documented when user uses `--help`.

Reply:
Yup! I think it should be noted here: https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/core/core_plugin.py#L319-L321

Reply:
Done, PTAL.  I reworded help text across several flags to try to make it clearer.